### PR TITLE
algebraicpetri model for scenario 2

### PR DESCRIPTION
**Pull request by olynch**

### CODE REVIEW

File: docs/src/scenarios/scenario2.md [hunk 1]
```diff
@@ -6,71 +6,48 @@
...
-seirhd_vax = stratify_typed(
-  seirhd_typed=>[[:strata],[:strata],[:strata],[:strata],[:strata],[]],
-  Vax_aug_typed=>[[:disease,:infect],[:disease,:infect]],
-  types′)
-
-@assert is_natural(seirhd_vax)
+# The names for the typed product are tuples; here we combine them with an underscore
+seirhd_vax = map(
+  seirhd_vax_typed.dom,
+  Name=t -> Symbol(string(t[1]) + "_" + string(t[2]))
```
Comment (Keno) [inline docs/src/scenarios/scenario2.md:49]: string concat is `*` or just `string(a, b, c)`.

Comment (ChrisRackauckas) [inline docs/src/scenarios/scenario2.md:49]:
```suggestion
  Name=t -> Symbol(string(t[1]) * "_" * string(t[2]))
```

Comment (olynch) [reply docs/src/scenarios/scenario2.md]: I fixed this when I was copy-pasting into the repl, but forgot to copy it back into the code...

**Comment by ChrisRackauckas** [comment]
Does this need to bump lower bounds?

**Pull request closed without merging**